pysat/tests/test_orbits.py

```diff
@@ -3,6 +3,8 @@
 import numpy as np
 
 import pandas as pds
+# Import below is needed for tests on __repr__
+from pandas import Timedelta # noqa
```

Comment:
```suggestion
from pandas import Timedelta # noqa: F401
```
We should also specify which flake8 rule we're bypassing here.  It should be "module is imported but unused"